hardware/mcp3008/pimcp3008.js

```diff
@@ -1,12 +1,12 @@
 
 module.exports = function(RED) {
     "use strict";
-    var fs = require('fs');
+    var fs2 = require('fs');
     var allOK = false;
     var mcpadc;
     // unlikely if not on a Pi
     try {
-        var cpuinfo = fs.readFileSync("/proc/cpuinfo").toString();
+        var cpuinfo = fs2.readFileSync("/proc/cpuinfo").toString();
```

Comment:
Likewise - why won't the original be ok ?

Reply:
The line can remain also original.